bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/seqcomp/GhostSeqRepeatFlatObs.java

```diff
@@ -65,7 +67,7 @@ public int getLampCount() {
         return lamps.size();
     }
 
-    public Set<Lamp> getLamps() {
+    public TreeSet<Lamp> getLamps() {
```

Comment:
Why not return the more generic type?

Reply:
I have no objection returning `Set`, but you mentioned in the last PR that order might be important (and it is stored and returned as `TreeSet`), so I figured I would carry through the order preservation.

Reply:
(I was lifting this code from `SeqRepeatFlatObs`, which uses `TreeSet`.)